scenario.py

```diff
@@ -157,6 +157,12 @@ def __init__(self, params, experiment_path, scenario_id=1, n_repeat=1, is_dry_ru
         else:
             self.is_early_stopping = True  # default
 
+        # Model from previous coalition used to initialise model
+        try:
+            self.init_model_from = params["init_model_from"]
+        except:
+            self.init_model_from= ""
+
```

Comment:
@celinejacques is that a Python best practice? If so, it is really cool because I wasn't too fond of our `if ... else ...` for all config parameters. Just tell me, and I can take care of refactoring all those with your `try ... except ...` approach!

Reply:
I'm not sure that it is a Python best practice and I'm assuming that it depends on each case. Here, I found it usefull. 
@RomainGoussault @natct10 what do you think?

Reply:
I would tend to say that `except` is cool to catch an exception, so here if/else would be sufficient, but I don't have objections

Reply:
@natct10 I think here we are facing a potential exception: if the parameter is not in `params`. No?

Reply:
Yep!